src/main/java/org/gitlab4j/api/models/Group.java

```diff
@@ -295,6 +303,11 @@ public Group withSharedProjects(List<Project> sharedProjects) {
         return this;
     }
 
+    public Group withRunnersToken(String runnersToken) {
```

Comment:
I don't see in [Groups documentation](https://docs.gitlab.com/ee/api/groups.html) a reference to a creation of a group with a specified `runners_token`. Are you sure this method is useful?

Reply:
Since the token is only returned from the api and can't be set over the api, so I will remove the builder method.